Vagrantfile

```diff
@@ -1,27 +1,30 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
-# Provision root account with the insecure Vagrant ssh key.
```

Comment:
I think that this is already done by vagrant and we don't need to manually add it to the vagrant-vm